instrumentation/que/lib/opentelemetry/instrumentation/que/instrumentation.rb

```diff
@@ -10,7 +10,7 @@ module Que
       # The Instrumentation class contains logic to detect and install the Que
       # instrumentation
       class Instrumentation < OpenTelemetry::Instrumentation::Base
-        MINIMUM_VERSION = Gem::Version.new('1.0.0.beta4')
+        MINIMUM_VERSION = Gem::Version.new('1.2.0')
```

Comment:
So that I'm clear here, any users who were using this instrumentation with versions earlier than this will no longer be able to use this instrumentation. I understand the need to upgrade from a beta version but this now skips past 1.0 and 1.1, is that ok?

Reply:
Yes, so users would need to be on at least que `v1.2`. The changlogs for 1.1 and 1.0.0.beta5 only has bug fixes and improvements, nothing that would seem to break changes between version jumps. 

[v1.0.0.beta5 changelog](https://github.com/que-rb/que/blob/master/CHANGELOG.md#100beta5-2021-12-23)
[v1.1 changelog](https://github.com/que-rb/que/blob/master/CHANGELOG.md#110-2022-02-21)